DevOps/docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3'
+services:
+  jenkins_server:
+    image: "registry.jala.info/devops/jenkins/jenkins:2.150.3-alpine"
+	ports:
```

Comment:
I think there is an error for indentation.

Reply:
fixed